docs/docs/guides/web3_providers_guide/index.md

```diff
@@ -0,0 +1,173 @@
+---
+sidebar_position: 1
+sidebar_label: 'Providers'
+---
+
+# web3.js Providers Guide
+
+Connecting to a chain happens through a provider. You can pass the provider to the constructor as in the following example:
+
+```ts
+import { Web3 } from `web3`
+
+const web3 = new Web3(/* PROVIDER*/);
+
+// calling any method that interact with the network would involve using the early passed provider.
+await web3.eth.sendTransaction({
+    from,
+    to,
+    value,
+});
+```
+
+The created Web3 instance will use the passed provider to interact with the blockchain network. This interaction happen when sending a request and receiving the response, and when possibly listen to provider events (if the provider support this).
+
+## Providers Types
+
+Actually, the provider could be any of the following:
```

Comment:
```suggestion
The provider could be any of the following:
```

Reply:
I think this is a valid and good use of `Actually`.
https://www.mmmenglish.com/2019/11/08/4-ways-to-use-actually/

Reply:
This is a nitpick by me, feel free to ignore. 
`Actually` is valid for sure, using it to start a sentence for documentation is a bit uncommon. I think you can convey the same information without it

Reply:
Thanks Alex, 
Yes, it does not add any information and we can still provide the same information without it. However, such a word plays a role in bringing attention and making the documentation less boring I think :smile: